src/kubernetes-ingress-controller/guides/using-udpingress.md

```diff
@@ -1,353 +1,328 @@
 ```
+{% endnavtab %}
+{% endnavtabs %}
 
-Watch the services using `kubectl get services` and wait for the `LoadBalancer` service to be ready for the Gateway.
-Once the service up, {{site.base_gateway}} is ready to serve UDP traffic on the external port `9999`
-using `UDPIngress` resources.
-
-## Deploying UDPIngress
+[echoserver-udp](https://hub.docker.com/r/cilium/echoserver-udp) is a simple
+test server that accepts UDP TFTP requests and returns basic request
+information. As curl supports TFTP, it is a convenient option for
+testing UDP routing.
 
-Now that {{site.base_gateway}} is listening on `9999`, you can create a `UDPIngress` resource which will attach
-the CoreDNS service to that port so you can make DNS requests to it from outside the cluster.
+## Route UDP traffic
 
-Save the following file as `coredns-udpingress.yaml`:
+Now that {{site.base_gateway}} is listening on `9999` and the test application
+is running, you can create UDP routing configuration that proxies traffic to
+the application:
 
-```yaml
-apiVersion: configuration.konghq.com/v1beta1
+{% navtabs api %}
+{% navtab Ingress %}
+{% navtabs codeblock %}
+{% navtab Command %}
+```bash
+echo "apiVersion: configuration.konghq.com/v1beta1
 kind: UDPIngress
 metadata:
-  name: minudp
-  namespace: udpingress-example
+  name: tftp
+  namespace: udp-example
   annotations:
     kubernetes.io/ingress.class: kong
 spec:
   rules:
   - backend:
-      serviceName: coredns
-      servicePort: 53
+      serviceName: tftp
+      servicePort: 9999
     port: 9999
+" | kubectl apply -f -
 ```
-
-This configuration binds the {{site.base_gateway}} port `9999` to the `Service` port `53` for our DNS server.
-
-Apply the `coredns-udpingress.yaml` manifests:
-
-```shell
-$ kubectl apply -f coredns-udpingress.yaml
+{% endnavtab %}
+{% navtab Response %}
+```text
+udpingress.configuration.konghq.com/tftp created
 ```
+{% endnavtab %}
+{% endnavtabs %}
+{% endnavtab %}
+{% navtab Gateway APIs %}
+{% navtabs codeblock %}
+{% navtab Command %}
+```bash
+echo "apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: UDPRoute
+metadata:
+  name: tftp
+  namespace: udp-example
+spec:
+  parentRefs:
+  - name: kong
+    namespace: default
+  rules:
+  - backendRefs:
+    - name: tftp
+      port: 9999
+" | kubectl apply -f -
+```
+{% endnavtab %}
+{% navtab Response %}
+```text
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endnavtab %}
+{% endnavtabs %}
 
-You can now make DNS requests via {{site.base_gateway}}.
-
-## Verification
-
-Now that setup is complete, all that's left to do is verify that everything is working
-by making a DNS request to our CoreDNS server.
+This configuration routes traffic to UDP port `9999` on the
+{{site.base_gateway}} proxy to port `9999` on the TFTP test server.
 
-> **Note:** This example assumes you have the `dig` command available on your local
- system. If you don't, refer to your operating system documentation for a similar DNS
- lookup tool.
+## Test the configuration
```

Comment:
Ditto here.

Reply:
Left this one as-is, UDP only has one type of config.